tests/pubsub/testmcache.nim

```diff
@@ -6,9 +6,8 @@ import
   ../../libp2p/
     [peerid, crypto/crypto, protocols/pubsub/mcache, protocols/pubsub/rpc/message]
 
-var rng = newRng()
-
 proc randomPeerId(): PeerId =
+  var rng = newRng()
```

Comment:
depending on the os, creating a new rng for every peer id might exhaust the randomness supply and stall the test